src/types.ts

```diff
@@ -72,7 +72,8 @@ export interface HoneycombOptions extends Partial<WebSDKConfiguration> {
   serviceName?: string;
 
   /** The sample rate used to determine whether a trace is exported.
-   * This must be a whole number greater than 1. Only 1 out of every `sampleRate` traces will be randomly selected to be sent.
+   * This must be a whole positive number. Only 1 out of every `sampleRate` traces will be randomly selected to be sent.
+   * Set to 0 to drop everything.
```

Comment:
This might not be super common but it's an easy lever to pull so it's nice to have.

Reply:
if we drop everything, do we still get logs if `debug` is true? like could someone set it to zero to see what events _would_ be sent but not actually send events over the wire?

Reply:
Ooh excellent question. I'll double check to confirm, but I'm pretty sure it _won't_ log the events because the debug logger is intended to show events that will be sent, and this stops them from getting that far. This is more intended as an emergency lever, or closer to a no-op, to quickly disable further processing and exporting of telemetry.

Reply:
🤔  Something to consider though... I suppose we might like a ConsoleSpanExporter easy-button. Since debug _only_ logs if something is successfully exported, a "dev" option could ignore everything else and just always send to stdout and never try to export elsewhere, for exactly what you suggest here. I'll make an issue for consideration.

Reply:
- https://github.com/honeycombio/honeycomb-opentelemetry-web/issues/73